README.md

```diff
@@ -37,48 +55,47 @@ provider "k8s" {
 ```
 
 **WARNING:** Configuration from the variable will be recorded into a temporary file and the file will be removed as
-soon as call is completed. This may impact performance if the code runs on a shared system because
-and the global tempdir is used.
+soon as call is completed. This may impact performance if the code runs on a shared system because the global
+tempdir is used.
 
-The k8s Terraform provider introduces a single Terraform resource, a `k8s_manifest`. The resource contains a `content` field, which contains a raw manifest.
+### Resource Definition
 
-```hcl
-variable "replicas" {
-  type    = "string"
-  default = 3
-}
+An example of the k8s_manifest might look like.
 
-data "template_file" "nginx-deployment" {
-  template = "${file("manifests/nginx-deployment.yaml")}"
+The terraform resource definition should include:
 
-  vars {
-    replicas = "${var.replicas}"
-  }
-}
+* `name` of the resource
+* `namespace` to be deployed to
+* `content` resource to be deployed
```

Comment:
Doesn't it require `kind` too? and `namespace` is only needed for resources which are namespaced so not cluster resources.

Reply:
oops, forgot I removed that. Added your description for namespace